packages/lesswrong/components/posts/PostsPage/PostsPageDate.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { registerComponent, Components } from 'meteor/vulcan:core';
+import { withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import { ExpandedDate } from '../../common/FormatDate.jsx';
+import classNames from 'classnames';
+
+const styles = theme => ({
+  date: {
+    marginLeft: 20,
+    display: 'inline-block',
+    color: theme.palette.grey[600],
+    fontSize: theme.typography.body2.fontSize,
+  },
+  mobileDate: {
+    [theme.breakpoints.up('md')]: {
+      display:"none"
+    }
+  },
+  desktopDate: {
+    whiteSpace: "no-wrap",
+    [theme.breakpoints.down('sm')]: {
+      display:"none"
+    }
+  },
+});
+
+const PostsPageDate = ({ post, hasMajorRevision, classes }) => {
+  const { FormatDate, PostsRevisionSelector } = Components;
+  
+  const tooltip = (<div>
+    <div>Posted on <ExpandedDate date={post.postedAt}/></div>
+    
```

Comment:
pretty sure html will remove this whitespace, but not quite sure why it's there in the first place

Reply:
Right, this blank line doesn't do anything. It's an arbitrary code-style thing.